### DISCUSSION TITLE
AdHoc::on_attach(..) not present in v0.5-rc - Alternative?

### DISCUSSION THREAD

**Discussion by skerr0** [rwf2/Rocket#2123] [Questions]
I was trying to add a fairing via AdHoc. Within the inline function, I wanted to access the rocket config to access some custom values in this function.

 Documentation has some examples using AdHoc::on_attach(), which look appropriate for this purpose. This function is not available in rc0.5-rc. What is the recommended way of accessing the rocket configuration within an AdHoc fairing?

**Comment by SergioBenitez** [comment]
Use `on_ignite` or `on_liftoff`.
